tutorials/tour/basics.md

```diff
@@ -90,33 +91,33 @@ println({
 
 ## Functions
 
-Functions are expressions with parameters.
+Functions are expressions that take parameters and compute a result.
```

Comment:
In a function application, are arguments applied to a function or the function to the arguments? Params and args can also be values or types. Functions also reject args when an application fails to typecheck.